packages/clerk-js/src/ui/elements/Select.tsx

```diff
@@ -369,11 +369,7 @@ export const SelectButton = (
 
   let show: React.ReactNode = children;
   if (!children) {
-    show = selectedOption ? (
-      buttonRenderOption(selectedOption)
-    ) : (
-      <Text sx={t => ({ opacity: t.opacity.$inactive })}>{placeholder}</Text>
```

Comment:
@desiprisg is the inactive opacity replaced with a color?

Reply:
I'm not sure I made this change. I'll revert though.